cmd/vulcan-seekret/manifest.toml

```diff
@@ -2,4 +2,4 @@ Description = "Finds leaked secrets in a Git repository using Seekret"
 Timeout = 600
 AssetTypes = ["GitRepository"]
 RequiredVars = ["GITHUB_ENTERPRISE_ENDPOINT", "GITHUB_ENTERPRISE_TOKEN"]
-Options = '{"depth": 1, "branch": "master"}'
+Options = '{"depth": 1, "branch": ""}'
```

Comment:
I maybe put here a comment stating that, if the branch option is empty,  the check will scan the default branch of the repo.